usagov/usagov-2021
# USAGOV-1232-Narrow-Footer-On-Spanish-Elected-Officials

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1232

## Description

I compared the differences in the pagesource between usa.gov/es/funcionarios-electos and cms.usa.gov/es/funcionarios-electos. The biggest discrepancy that I could find was an unclosed script tag which appears to be duplicated so I removed it.

I attempted to test this locally but the issue only appears on the static site and I was not able to generate the static pages locally. So please deploy to dev where we can see if it fixes the issue on the dev static site.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


I attempted to test this locally but the issue only appears on the static site and I was not able to generate the static pages locally. So please deploy to dev where we can see if it fixes the issue on the dev static site.
* Deploy this change to the dev environment
* Visit beta-dev.usa.gov/es/funcionarios-electos
* Confirm that the footer fills the width of the viewport

### Requires New Config

### Requires New Content

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
